scripts/sct_image.py

```diff
@@ -19,9 +19,9 @@
 
 import spinalcordtoolbox.image as msct_image
 from spinalcordtoolbox.image import Image, concat_data
-from spinalcordtoolbox.utils import Metavar, SmartFormatter
-from nibabel                 import Nifti1Image
-from nibabel.processing      import resample_from_to
+from spinalcordtoolbox.utils import Metavar, SmartFormatter, init_sct, run_proc
+from nibabel import Nifti1Image
+from nibabel.processing import resample_from_to
```

Comment:
since you're at it, maybe you can move nibabel imports above SCT imports (for PEP8)?

Reply:
Sure. I'll compile a list in the parent issue and add this there.